docs/reference/commandline/daemon.md

```diff
@@ -188,9 +205,9 @@ Currently supported options of `devicemapper`:
     10 GB of space on the pool. However, the filesystem will use more space for
     the empty case the larger the device is.
 
-     **Warning:** This value affects the system-wide "base" empty filesystem
-     that may already be initialized and inherited by pulled images. Typically,
-     a change to this value will require additional steps to take effect:
+    **Warning**: This value affects the system-wide "base" empty filesystem
```

Comment:
Actually, this information isn't appropriate for a Warning or a Note.  It should just be part of the text --- which differed between the two. Sync'd. Thanks for the catch.